pkg/smartcontract/manifest/standard/nep24.go

```diff
@@ -0,0 +1,47 @@
+package standard
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+)
+
+// Nep24 is a NEP-24 Standard for NFT royalties.
+var Nep24 = &Standard{
+	Manifest: manifest.Manifest{
+		ABI: manifest.ABI{
+			Methods: []manifest.Method{
+				{
+					Name: "royaltyInfo",
+					Parameters: []manifest.Parameter{
+						{Name: "tokenId", Type: smartcontract.ByteArrayType},
+						{Name: "royaltyToken", Type: smartcontract.Hash160Type},
+						{Name: "salePrice", Type: smartcontract.IntegerType},
+					},
+					ReturnType: smartcontract.ArrayType,
+					Safe:       true,
+				},
+			},
+		},
+	},
+	Required: []string{manifest.NEP11StandardName},
+}
+
+// RoyaltiesTransfer an event that MUST be emitted by marketplaces that support the `royaltyInfo` method.
+var RoyaltiesTransfer = &Standard{
```

Comment:
> RoyaltiesTransfer an event that MUST

Grammar. Ans it's not an event, make it similar to the reference: https://github.com/nspcc-dev/neo-go/blob/a4633ce2c701f8ef148f43132c9229d964b80576/pkg/smartcontract/manifest/standard/payable.go#L8-L9

> var RoyaltiesTransfer 

Rename it to `Nep24Payable`, make a separate constant for this standard. Add it to the list of standards check.

Reply:
also added `Nep24Payable` to binding.Generate as `mfst.ABI.Events = dropStdEvents(mfst.ABI.Events, standard.Nep24Payable)` in case of `ctr.IsNep24 = true`. Do we need a separate test for it?

Reply:
> Do we need a separate test (new smartcontract in testdata) for it?

Would be nice, and this contract shouldn't be large in fact. A single method is enough for the testing purposes.

Reply:
Test is still needed.